autogpt_platform/backend/backend/integrations/oauth/google.py

```diff
@@ -17,6 +21,11 @@ class GoogleOAuthHandler(BaseOAuthHandler):
 
     PROVIDER_NAME = "google"
     EMAIL_ENDPOINT = "https://www.googleapis.com/oauth2/v2/userinfo"
+    DEFAULT_SCOPES = [
```

Comment:
is this getting access to users email? What does it entail exactly?

Reply:
We can read your emails!

Reply:
Oh sorry this is for naming the token if I remember correctly. The default scopes is required because there’s some info we need like your email so we can name the token